python/lsst/ts/scheduler/scheduler_csc.py

```diff
@@ -951,6 +951,14 @@ async def _queue_one_script(
             The uid of the block.
         script : `ObservingScript`
             Observing script to queue.
+        block_name : `str`
+            Name of the block (e.g. BLOCK-1).
+        start_block : `bool`
+            Start a new block?
+            Should be `True` for the first script of a block,
+            `False` otherwise.
```

Comment:
It looks like this is not checked in the code. What happens if it is False for the first block or True for any other block than the first? Please either update the doc string indicating that or implement a check.